book/content/reproducibility/04/barriers.md

```diff
@@ -40,9 +40,18 @@ This chapter outlines some of those barriers, and a couple of suggestions to get
 
 *replace this text with the content of barrier 7*
 
-## Barrier 8
-
-*replace this text with the content of barrier 8*
+## Big data and complex computational infrastructure
+
+Big data is conceptualised in different ways by different researchers. 
+The traditional understanding of big data focuses on complexity, variety of data sources, volume and velocity of the data. 
+It is extremely difficult to control to ensure consistency of data between the experiments. 
```

Comment:
```suggestion
Although there are ways to set random seeds and take snapshots of a dataset at a particular moment in time, it can be difficult to have identical data across different runs of an analysis pipeline.
This is particularly relevant in the context of tools for parallel computing.
```